src/Events.php

```diff
@@ -166,6 +168,7 @@ protected function possibly_delete_notes() {
 		NavigationFeedback::delete_if_not_applicable();
 		NavigationFeedbackFollowUp::delete_if_not_applicable();
 		SetUpAdditionalPaymentTypes::delete_if_not_applicable();
+		PaymentsRemindMeLater::delete_if_not_applicable();
```

Comment:
It seems this was missed to be implemented in the note class. In bridge, its [here](https://github.com/Automattic/wc-calypso-bridge/pull/756/files#diff-942cc9b727aa516c25b7a48756ca826ccd8b5a99b7031b397507aa0cb001aa60R83-R86)

Reply:
@ilyasfoo Good catch! 👍  I found there is an additional `()` behind the `PaymentsRemindMeLater::possibly_add_note();`. I just fixed this in https://github.com/woocommerce/woocommerce-admin/pull/8085/commits/fcd944771a9db106645c69d1c831242e34be6fd4 I've confirmed on my local. Please test again. Thanks!